pilot/platform/cloudfoundry/cloudfoundry_suite_test.go

```diff
@@ -1,27 +1,115 @@
-// Copyright 2017 Istio Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 package cloudfoundry_test
 
 import (
 	"testing"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
 )
 
-func TestCloudFoundry(t *testing.T) {
+func TestCloudfoundry(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "Cloud Foundry Suite")
+	RunSpecs(t, "Cloudfoundry Suite")
 }
+
+// Declarations for Ginkgo DSL
+type Done ginkgo.Done
```

Comment:
This was not really the intent of removing the dot imports - the whole point is to be able to see where some function is coming from. Just revert this commit and go back to dot imports since the follow on will be removing ginkgo and gomega anyway.

Reply:
gotcha.  done.